tests/src/test/java/alluxio/shell/command/MountCommandTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.shell.command;
+
+import alluxio.AlluxioURI;
+import alluxio.client.WriteType;
+import alluxio.client.file.options.CreateDirectoryOptions;
+import alluxio.shell.AbstractAlluxioShellTest;
+import alluxio.util.io.BufferUtils;
+import alluxio.util.io.PathUtils;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Tests for mount command.
+ */
+public final class MountCommandTest extends AbstractAlluxioShellTest {
+  @Rule
+  public TemporaryFolder mFolder = new TemporaryFolder();
+
+  private void checkMountPoint(AlluxioURI mountPoint, String ufsPath) throws Exception {
+    // File in UFS can be read in Alluxio
+    AlluxioURI testFile = mountPoint.join("testFile");
+    generateRelativeFileContent(PathUtils.concatPath(ufsPath, "testFile"),
+        BufferUtils.getIncreasingByteArray(10));
+    Assert.assertTrue(fileExists(testFile));
+    // Dir in Alluxio can be persisted to UFS
+    AlluxioURI testDir = mountPoint.join("testDir");
+    mFileSystem.createDirectory(testDir,
+        CreateDirectoryOptions.defaults().setWriteType(WriteType.CACHE_THROUGH));
+    Assert.assertTrue(fileExists(testDir));
+    Assert.assertTrue(Files.exists(Paths.get(ufsPath, "testDir")));
+  }
+
+  @Test
+  public void mount() throws Exception {
+    AlluxioURI mountPoint = new AlluxioURI("/mnt");
+    String ufsPath = mFolder.getRoot().getAbsolutePath();
+    Assert.assertEquals(0, mFsShell.run("mount", mountPoint.toString(), ufsPath));
+    checkMountPoint(mountPoint, ufsPath);
+  }
+
+  @Test
+  public void mountWithWrongArgs() throws Exception {
```

Comment:
Can you add some tests with correct args? (tricky correct args too, like `=` or ` ` in values)